File: R/rdv.r
#' @export 
make.rdv <- function(lieu, H, M, S) {
  a <- mkRDV(lieu, H, M, S)
  class(a) <- "rdv"
  a
}

#' @export 
print.rdv <- function(x, ...) {
  cat("Rendez vous à ", getLieu(x), " à ", getHeure(x), "\n")
}

File: src/rdv.cpp
#include "rdv.h"

std::ostream & operator<<(std::ostream & out, const rdv & a) {
  out << a.lieu << " à " << (heure) a << "\n";
  return out;
}

// out << a << b << c
// ((out << a) << b) << c